pkgs/top-level/all-packages.nix

```diff
@@ -5828,6 +5828,10 @@ with pkgs;
 
   mousetweaks = callPackage ../applications/accessibility/mousetweaks { };
 
+  moq-relay = callPackage ../by-name/mo/moq-relay/package.nix {
+    inherit libressl pkg-config;
+  };
+
```

Comment:
IIRC, this isn't required for packages in ``by-name``.

Reply:
fixed